custom/enikshay/ucr/reports/dmc_lab_summary.json

```diff
@@ -0,0 +1,180 @@
+{
+  "domains": [
+    "enikshay",
+    "enikshay-reports-qa"
+  ],
+  "server_environment": [
+    "enikshay",
+    "softlayer"
+  ],
+  "data_source_table": "test_2b_v5",
+  "report_id": "dmc_lab_summary",
+  "config": {
+    "description": "",
+    "title": "DMC Lab Summary",
+    "sort_expression": [],
+    "visible": true,
+    "configured_charts": [],
+    "aggregation_columns": [],
```

Comment:
is this report tested at all, my understanding was that you need something in here for a ucr to work, and properly aggregate data

Reply:
@calellowitz Yes this report was tested, if we have aggregation_columns empty then in the report we have only one record, and this is expected behavior.

Reply:
sounds good